project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.0
+sbt.version=1.2.8
```

Comment:
Was this version update intentional?

Reply:
yes. had to upgrade because of metrics PR might as well update to latest.